examples/gallery/grid/grdview_surface.py

```diff
@@ -45,6 +46,7 @@ def ackley(x, y):
     surftype="s",
     cmap="roma",
     perspective="135/30",
+    I="+a45",
```

Comment:
@liamtoney This is a nice improvement to the gallery. However, we prefer to to use long-form arguments (e.g., `shading="+a45"`, instead of short-form arguments (e.g., `I="+a45"`). See https://github.com/GenericMappingTools/pygmt/issues/473 for the reason, and https://github.com/GenericMappingTools/pygmt/pull/474 for our recent efforts.

However, the `grdview()` function doesn't provide a long-form alias for the **-I** option. So we need to do that first. The change is simple, just add `I="shading"` to the `use_alias` decorator of the `grdview()` function. You can take the `grdimage()` function as an example.

Reply:
Right. Would that need to be a separate PR, or can that happen in this one?

Reply:
It's a tiny change. I think it's OK to do it in this PR.